angr/analyses/cfg/indirect_jump_resolvers/jumptable.py

```diff
@@ -657,16 +691,7 @@ def filter(self, cfg, addr, func_addr, block, jumpkind):
                 l.warning('JumpTableResolver does not support P-Code IR yet; CFG may be incomplete.')
             return False
 
-        if is_arm_arch(self.project.arch):
-            # For ARM, we support both jump tables and "call tables" (because of how crazy ARM compilers are...)
-            if jumpkind in ('Ijk_Boring', 'Ijk_Call'):
-                return True
-        else:
-            # For all other architectures, we only expect jump tables
-            if jumpkind == 'Ijk_Boring':
-                return True
-
-        return False
+        return jumpkind in ('Ijk_Boring', 'Ijk_Call')
```

Comment:
Nit: Use a set instead of a list.

```
In [1]: %timeit 'Ijk_Call' in ('Ijk_Boring', 'Ijk_Call')
25.9 ns ± 0.363 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [2]: %timeit 'Ijk_Call' in {'Ijk_Boring', 'Ijk_Call'}
20 ns ± 0.311 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Reply:
Fixed